hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Datanode Chunk Read_Write Dashboard.json

```diff
@@ -15,7 +15,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "10.4.2"
```

Comment:
Naive question: Would other grafana servers be able to import dashboard generated from newer grafana.

Reply:
Good question. Even with the local setup the docker imaged pulled down is the latest. I used the older version `10.4.2` of the docker image and the dashboards were rendered without errors. This will be a hard thing for us to keep track off across versions.

Reply:
I tried on an older version `Version 7.3.6 (commit: NA, branch: master)` and a lot of the dashboards don't really work.. but I think it is ok for us to continue with the latest version.